puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_deb_coreproject.yaml

```diff
@@ -32,7 +32,7 @@
           values:
           - debian
     execution-strategy:
-      combination-filter: '((onlyos == "all") || onlyos == os) && !(arch == "armv7" && (os == "trusty")) && !(arch == "armv8" && os == "trusty") && !((version == "1.14" || version == "1.15") && os == "stretch")'
+      combination-filter: '((onlyos == "all") || onlyos == os) && !(os == "trusty" && repo == "develop") && !(arch == "armv7" && (os == "trusty")) && !(arch == "armv8" && os == "trusty") && !((version == "1.14" || version == "1.15") && os == "stretch")'
```

Comment:
1. Prefer testing `version` as repo may vary when building against user branches in PRs etc.
2. Prefer listing which releases it should build for (realistically, only 1.15 now), then it won't need updating later.

(Copy the Stretch conditional.)